libyul/optimiser/DataFlowAnalyzer.h

```diff
@@ -171,6 +176,8 @@ class DataFlowAnalyzer: public ASTModifier
 
 	std::unordered_map<YulString, YulString> m_storage;
 	std::unordered_map<YulString, YulString> m_memory;
+	/// If m_keccak[s, l] = y then y := keccak256(s, l) occurs in the code.
+	std::map<std::pair<YulString, YulString>, YulString> m_keccak;
```

Comment:
Class docstring should be updated to mention keccak. Looks like [`DataflowAnalyzer`](https://docs.soliditylang.org/en/latest/internals/optimizer.html#dataflowanalyzer) is also in the docs so I'd update that too.